Earthfile

```
@@ -235,7 +240,8 @@ CACHE_WARMUP:
          .evergreen/ensure-ninja.sh \
          /T/
     RUN bash /T/ensure-cmake.sh
-    RUN env NINJA_EXE=/usr/local/bin/ninja \
+    ARG ninja_version
+    RUN env NINJA_VERSION=$ninja_version NINJA_EXE=/usr/local/bin/ninja \
```

Recommend giving this a default value and making it a persistent environment variable like:

```suggestion
    ARG ninja_version = "x.y.z"
    ENV NINJA_VERSION=$ninja_version
    RUN env NINJA_EXE=/usr/local/bin/ninja
```

---

You are probably correct here. Doubling the default value is probably not great.

---

Commit incoming with a modified version of this (based on our conversation in Slack).